packages/api/models/granules.js

```diff
@@ -32,6 +33,15 @@ const {
 const Rule = require('./rules');
 const granuleSchema = require('./schemas').granule;
 
+const translateGranule = async (granule) => {
+  if (isNil(granule.files)) return granule;
+
+  return {
+    ...granule,
+    files: await buildDatabaseFiles({ files: granule.files })
```

Comment:
Does this have any problems if you `get` a granule that is stored in the new schema? Specifically with an eye towards the error thrown by `getSource` if `file.path` is missing, which no longer exists in the new schema?

Reply:
I did a quick search and couldn't find `getSource()`, can you post a link to it?

Reply:
My bad, I meant `setSource` in `FileUtils`: https://github.com/nasa/cumulus/blob/CUMULUS-1139-master/packages/api/lib/FileUtils.js#L67

Reply:
I see now that there would be no `providerURL` so `setSource` would return without going into `buildFileSourceURL`, which would throw the error otherwise. I withdraw the question. Thanks.